web/pgadmin/utils/driver/psycopg3/connection.py

```
@@ -447,7 +447,7 @@ def _set_role(self, manager, cur, conn_id, **kwargs):
             role = manager.role
 
         if is_set_role:
-            _query = "SELECT rolname from pg_roles WHERE rolname = {0}" \
+            _query = "SELECT rolname from pg_catalog.pg_roles WHERE rolname = {0}" \
                      "".format(self.qtLiteral(role, self.conn))
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)